docs/content/dagster-plus/deployment/azure/blob-compute-logs.mdx

```diff
@@ -0,0 +1,119 @@
+# Storing compute logs in Azure Blob Storage/Azure Data Lake Storage
+
+In this guide, we'll walk through how to store compute logs in Azure Blob Storage or Azure Data Lake Storage. This guide assumes you have already set up an Azure Kubernetes Service (AKS) agent and deployed user code in Azure Container Registry (ACR).
+
+This guide focuses on using Azure Blob Storage, but the same steps should be applicable for Azure Data Lake Storage.
+
+If you have not yet set up an AKS agent, you can follow the [Deploy an Azure Kubernetes Service (AKS) agent guide](/dagster-plus/deployment/azure/aks-agent).
+If you have not yet deployed user code in ACR, you can follow the [Deploy user code in Azure Container Registry (ACR) guide](/dagster-plus/deployment/azure/acr-user-code).
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+- The Azure CLI installed on your machine. You can download it [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+- An Azure account with the ability to create resources in Azure Blob Storage or Azure Data Lake Storage.
+- An Azure container in Azure Blob Storage or Azure Data Lake Storage where you want to store logs.
```

Comment:
- Either the `quickstart_etl` module or any other code location successfully imported containing at least one job or asset to materialize with some logging verbosity included 

(see other comment about all_assets_job)